src/features/AddProduct/components/FileInput.js

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect, useRef } from "react";
+import styles from "./FileInput.module.css";
+import plusImg from "assets/images/ic_plus.svg";
+import XButton from "components/XButton";
+
+const FileInput = ({ label, name, value, onChange }) => {
+  const [preview, setPreview] = useState("");
+  const [isExist, setIsExist] = useState(false);
+
+  const inputRef = useRef();
+
+  const handleChange = (e) => {
+    const nextValue = e.target.files[0];
+    onChange(name, nextValue);
+  };
+
+  const handleUploadClick = (e) => {
+    if (value) {
+      e.preventDefault();
+      setIsExist(true);
+    }
+  };
+
+  const handleClearClick = (e) => {
+    e.preventDefault();
+    const inputNode = inputRef.current;
+    if (!inputNode) return;
+
+    inputNode.value = "";
+    onChange(name, null);
+    setIsExist(false);
+  };
+
+  useEffect(() => {
+    if (!value) return;
+
+    const nextPreview = URL.createObjectURL(value);
+    setPreview(nextPreview);
+  }, [value]);
+
+  return (
+    <div className={styles.container}>
+      <h3 className={styles.label}>{label}</h3>
+      <div className={styles.imageUploadContainer}>
+        <label
+          className={`${styles.imageUpload} ${styles.imageContainer}`}
+          htmlFor={name}
+          onClick={handleUploadClick}
+        >
+          <img src={plusImg} alt="이미지 등록" />
+          <p>이미지 등록</p>
+        </label>
+        <input
+          className={styles.input}
+          type="file"
+          accept="image/png image/jpeg"
```

Comment:
P1:
accept 속성을 여러개 주실때는 , 로 구분하셔야 합니다.

```suggestion
          accept="image/png, image/jpeg"
```

Reply:
P2:
accept 속성은 유저가 파일선택시 가능한 파일 확장자를 "제안"하는 역할을 합니다.
여기서 중요한점은 제한하는 것이 아니라는 것입니다. 유저는 해당 속성외의 파일을 업로드할 수 있습니다.
따라서 accept로 제안한 확장자 이외의 것을 업로드할 시 `onChange`같은 함수에서 이를 확인해 
업로드하지 않는 로직이 필요합니다.